src/python/director/segmentation.py

```diff
@@ -2529,47 +2532,33 @@ def segmentTableEdge(polyData, searchPoint, edgePoint):
     '''
 
     polyData, tablePoints, origin, normal = segmentTable(polyData, searchPoint)
-
     tableMesh = computeDelaunay3D(tablePoints)
-    origin, edges, wireframe = getOrientedBoundingBox(tableMesh)
-    origin = origin + 0.5*np.sum(edges, axis=0)
-
-
-    edgeLengths = np.array([np.linalg.norm(edge) for edge in edges])
-    axes = [edge / np.linalg.norm(edge) for edge in edges]
-
-    def findAxis(referenceVector):
-        refAxis = referenceVector / np.linalg.norm(referenceVector)
-        axisProjections = np.array([np.abs(np.dot(axis, refAxis)) for axis in axes])
-        axisIndex = axisProjections.argmax()
-        axis = axes[axisIndex]
-        if np.dot(axis, refAxis) < 0:
-            axis = -axis
-        return axis, axisIndex
-
-    tableXAxis, tableXAxisIndex = findAxis(searchPoint - edgePoint)
-    tableZAxis, tableZAxisIndex = findAxis([0,0,1])
-    tableYAxis, tableYAxisIndex = findAxis(np.cross(tableZAxis, tableXAxis))
-    assert len(set([tableXAxisIndex, tableYAxisIndex, tableZAxisIndex])) == 3
 
-    axes = tableXAxis, tableYAxis, tableZAxis
-    edgeLengths = edgeLengths[tableXAxisIndex], edgeLengths[tableYAxisIndex], edgeLengths[tableZAxisIndex]
-
-    edgeCenter = origin - 0.5 * axes[0]*edgeLengths[0] + 0.5*axes[2]*edgeLengths[2]
-    edgeLeft = edgeCenter + 0.5 * axes[1]*edgeLengths[1]
-    edgeRight = edgeCenter - 0.5 * axes[1]*edgeLengths[1]
-
-    t = getTransformFromAxes(axes[0], axes[1], axes[2])
-    t.PostMultiply()
-    t.Translate(edgeRight)
+    # TODO: replace this frame with view frame. (Currently viewframe is inverted on Valkyrie)
+    viewFrame = getLinkFrame("pelvis")# SegmentationContext.getGlobalInstance().getViewFrame()
```

Comment:
In the other comment I made, about the re-orientation code used for the drc finals terrain task, we used the stance frame as the reference frame.  The stance frame is usually very similar to the pelvis frame, though at a different z height obviously.  We have other tasks, too, where the stance frame is used to define "robot forward".  It's just slightly more generalized than using a named link frame such as pelvis, torso, or head.